overrides/components/EIC-related-contents.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "$veda-ui/react";
+import styled from "$veda-ui/styled-components";
+import { media, glsp } from "$veda-ui/@devseed-ui/theme-provider";
+import { stories } from "veda";
+import Hug from "$veda-ui-scripts/styles/hug";
+import {
+  Card,
+  CardList
+} from "$veda-ui-scripts/components/common/card";
+import { ContentsPropType } from './theme-card';
+
+const CardWrapper = styled(Hug)`
+  padding: ${glsp(1, 0)};
+  grid-column: 1/-1;
+`;
+
+const StyledCard = styled(Card)`
+  figure {
+    height: 200px;
+  }
+`
+
+const EvenCardList = styled(CardList)`
+  ${media.largeUp`
+    grid-template-columns: repeat(2, 1fr);
+  `}
+`
+
+export default function EICRelatedContents({ storyIds }:ContentsPropType) {
+  const relatedData = Object.keys(stories)
+  .map((key) => stories[key].data)
+  .filter((story) => storyIds.includes(story.id));
+
+  const cards = relatedData.map((t) => (
+    <li>
+      <StyledCard 
+        key={t.id}
+        linkLabel="View more"
+        linkTo={t.asLink?.url?? `/stories/${t.id}`}
+        title={t.name}
+        description={t.description}
+        imgSrc={t.media.src}
+        imgAlt={t.media.alt}
+      />
+    </li>
+  ))
+
+  const evenNumberCard = relatedData.length%2 == 0
```

Comment:
This actually confused me haha but I think what you are trying to do is that when there are an even amt of cards, the styling is a little different vs when there is an odd amt of cards, is this correct?

if so, can we rename `evenNumberCard` to `eventAmountOfCards` or `evenCardAmount` which I think makes more sense

Reply:
I moved the logic to the styled component level, hopefully, it is a bit more clear!